app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -257,8 +265,18 @@ private void doMenuAction(int menuItemId){
                 Intent intent = new Intent(MainActivity.this, SettingsActivity.class);
                 mDrawerLayout.closeDrawers();
                 startActivity(intent);
-
                 overridePendingTransition(R.anim.slide_in_left, R.anim.slide_out_left);
+            case R.id.nav_about:
+                new MaterialDialog.Builder(this)
+                        .iconRes(R.drawable.ic_launcher)
+                        .limitIconToDefaultSize()
+                        .title(String.format("%1$s", getString(R.string.app_name)))
+                        .content(getResources().getText(R.string.about_text))
+                        .positiveText(android.R.string.ok)
+                        .linkColorAttr(R.attr.colorPrimary)
+                        .positiveColorRes(R.color.primaryDarkColor)
+                        .show();
+                break;
```

Comment:
We want a style for the dialog as well. The ultimate goal is to be able to change the styles of the whole app from one place, so I think you should remove this and add the AppCompat Dialog and a corresponding style.